Easydict/NewApp/View/SettingView/Tabs/ServiceTab.swift

```diff
@@ -121,28 +121,29 @@ private struct ServiceItems: View {
     }
 }
 
+private class ServiceItemViewModel: ObservableObject {
+    @Published var isEnable = false
+
+    init(isEnable: Bool) {
+        self.isEnable = isEnable
+    }
+}
+
 @available(macOS 13.0, *)
 private struct ServiceItemView: View {
     let service: QueryService
 
     @EnvironmentObject private var viewModel: ServiceTabViewModel
 
-    private var enabled: Binding<Bool> {
-        .init {
-            service.enabled
-        } set: { newValue in
-            guard service.enabled != newValue else { return }
```

Comment:
This code is needed, we shouldn't post update notification if enable has no new value.

Reply:
The logic moved inside of `onChangeValue` modifier.